runtime/altair/src/xcm.rs

```diff
@@ -62,6 +63,10 @@ pub type Trader = (
 	FixedRateOfFungible<AirPerSecond, ToTreasury>,
 	FixedRateOfFungible<KUsdPerSecond, ToTreasury>,
 	FixedRateOfFungible<KsmPerSecond, ToTreasury>,
+	AssetRegistryTrader<
```

Comment:
To expand further on this one: regarding the XCM config logic looking up the asset-registry, we could in fact drop all the specific Trader and conversion logic associated with each individual natively handled token.

However, we can only do that once we have deployed the asset-registry _and_ registered all of those tokens in there.

I think that's definitely something we can do to clean up the code but I'd recommend we only do that after we have both deployed the `asset-registry` to `Altair` and XCM all-together to `Centrifuge` and have registered all the tokens in the asset-registry of each runtime respectively; this being so that we can then find a way of moving the XCM config to `runtime-commons` and simplify the process a lot.